src/components/containers/ArticleLists/ArticleLists.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Item, Icon } from 'semantic-ui-react';
+
+// components
+import Title from '../../presentationals/Title/Title';
+
+const f16 = {
+  fontSize: '16px',
+};
+
+class ArticleLists extends Component {
+  render() {
+    const { articles } = this.props;
+
+    return (
+      <div className="post-lists">
+        {articles.length === 0 ? (
```

Comment:
what do you think about writing !articles compared to articles.length === 0

Reply:
The article has an initial state of an empty array, `!articles` would mean that the variable is undefined